src/views/Hearing/Sidebar.js

```diff
@@ -102,17 +104,29 @@ class Sidebar extends React.Component {
               <div className="sidebar-section contents">
                 <h4><FormattedMessage id="table-of-content"/></h4>
                 <ListGroup>
-                  <ListGroupItem href="#hearing">
+                  {!isQuestionView && <ListGroupItem className={currentlyViewed === '#hearing' && 'active'} href="#hearing">
```

Comment:
There is already a branch containing the subsection links in the sidebar, as intended and specified in the wireframes: https://github.com/City-of-Helsinki/kerrokantasi-ui/pull/282

Apparently, Ville closed the branch with the intention of integrating this one, and forgot to introduce the code into this branch. However, this feature is needed.